### PR TITLE
chore(docs): update README with maintainers and good first issues

### PR DESCRIPTION
**Pull request by schalkneethling**
Adds info on project maintainers and link to good first issues.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
+## Project maintainers
+
+Should you have any questions regarding this project, please feel feel to @mention either @wbamberg or @schalkneethling
+
+## Good first issues
+
+Want to contribute to the interactive examples project? Here are a couple of good first issues to get you started. Thanks!
+
+[Good first issues](https://github.com/mdn/interactive-examples/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22example+needed%22+no%3Aassignee)
+
```
Comment (wbamberg) [inline README.md]: I think this should be in CONTRIBUTING.md :)

Comment (schalkneethling) [reply README.md]: Yes and no. I wanted these to be visible when you just look at the repo without clicking through to anything else. We could have it there as well, and perhaps other links for folks who have contributed before. Thoughts?

**Comment by schalkneethling** [comment]
@wbamberg Also added links to `CONTRIBUTING.md`

**Comment by schalkneethling** [comment]
Any thoughts on this one @wbamberg? Thanks

**Pull request merged**